runtime-optenv32/sdl+32/autobuild/defines

```diff
@@ -1,9 +1,12 @@
 PKGNAME=sdl+32
-PKGDES="Simple Directmedia Layer"
 PKGSEC=libs
-PKGDEP="x11-lib+32 alsa-lib+32"
-BUILDDEP="32subsystem"
+PKGDEP="aosc-aaa+32 x11-lib+32 alsa-lib+32"
+BUILDDEP="devel-base+32 nasm"
```

Comment:
Use `nasm+32` instead?

Reply:
This proves exactly that nasm+32 was not needed. Dropped.